scripts/rust-envvars

```diff
@@ -4,5 +4,8 @@
 export SNUBA_TEST_PYTHONPATH="$(python -c 'import sys; print(":".join(sys.path))')"
 export SNUBA_TEST_PYTHONEXECUTABLE="$(python -c 'import sys; print(sys.executable)')"
 
+# https://github.com/astral-sh/uv/issues/8821
+export PYTHONHOME="$(dirname $(dirname $(realpath $(which python))))"
```

Comment:
wait jk this only works with PYTHONHOME=~/.local/share/uv/python/cpython-3.11.11-macos-aarch64-none

not PYTHONHOME=/Users/josh/.local/share/sentry-devenv/pythons/3.11.11/python

wonder if devenv pythons are somehow different

Reply:
im kinda lost here so just going to fix forward with https://github.com/getsentry/snuba/pull/7368